src/config.rs

```diff
@@ -121,6 +121,21 @@ impl Target {
     }
 }
 
+impl Default for Target {
+    fn default() -> Self {
+        Self {
+            name: "".into(),
+            image: None,
+            uefi: false,
```

Comment:
IIUC uefi is fairly standard these days, so maybe this should be true?

Reply:
Ah ok, sounds good

Reply:
actually.... doing so would break existing configs because of: https://github.com/danobi/vmtest/blob/d3bc973fb7b94d247edfc07150f4b87785ddb346/src/vmtest.rs#L42

it would not impact one-liner usage though because it is set to false in https://github.com/danobi/vmtest/blob/d3bc973fb7b94d247edfc07150f4b87785ddb346/src/main.rs#L77